Explicitly exclude *.orig.port and *.beforesubst files left behind by our patch
to prevent getIpForModule() from breaking like this:

* Where:
Script '/usr/ports/pobj/ghidra-9.0.4/ghidra-Ghidra_9.0.4_build/gradleScripts/ip.gradle' line: 131

* What went wrong:
Execution failed for task ':CabExtract:ip'.
> No IP found for /usr/ports/pobj/ghidra-9.0.4/ghidra-Ghidra_9.0.4_build/GPL/CabExtract/build.gradle.orig.port in module: /usr/ports/pobj/ghidra-9.0.4/ghidra-Ghidra_9.0.4_build/GPL/CabExtract. Expression: (ip != null). Values: ip = null

Index: gradle/support/ip.gradle
--- gradle/support/ip.gradle.orig
+++ gradle/support/ip.gradle
@@ -154,6 +154,8 @@ def Map<String, List<String>> getIpForModule(Project p
 		exclude "**/*.vcxproj.user"
 		exclude '**/__pycache__'
 		exclude '**/*.egg-info'
+		exclude '**/*.orig.port'
+		exclude '**/*.beforesubst'
 	}
 	tree.each { file ->
 			String ip = getIp(p.projectDir, file)
